### ISSUE TITLE
Uploading a knowledge base document of 11M will report an error:  relation "embedding_vector_index_012c89d4 _31fa_477_8a00_372cefcd0344_nod" does not exist LINE 1: DELETE FROM embedding_V ector_index_012c89d4_31fa_477_8a00_3...

### ISSUE THREAD

**Issue by xuemusi** [langgenius/dify#7207]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

Version 0.6.16

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

My model ：
![image](https://github.com/user-attachments/assets/6daddfbf-1dc7-4ab8-9b70-4a7b7b6b17d5)
When I uploaded the PDF to the knowledge base, the above error was reported. Please assist in troubleshooting:
 ![image](https://github.com/user-attachments/assets/187496b4-8c03-4aad-b65c-7efc2faf33b6)

**Comment by crazywoola** [comment]
Please attach the logs in the description with proper format.

**Comment by xuemusi** [comment]
I have solved the problem. The main reason for this problem is that my pg did not install the vector plugin. 

Before installation: ![image](https://github.com/user-attachments/assets/0709bea4-fd3b-4283-8d99-2ea77de26a1f)

After installation via the following command line: 

` select control_extension('create','vector'); `

![image](https://github.com/user-attachments/assets/9461d4bd-e644-4780-869e-c8cdbea8a745)

Finally, I successfully uploaded the file.

 ![image](https://github.com/user-attachments/assets/52c81e04-24f0-484d-921b-0c3bf62adcd9)
